### PR TITLE
created simple-ca-for-ssh/de

### PR DESCRIPTION
**Pull request by pekab0**
I have read and understood the Contributor's Certificate of Origin
available at the end of https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Paul Ludwig paul.ludwig@hetzner.com

### CODE REVIEW

**Comment by n-se** [comment]
Sorry for tagging wrong and not reviewing yet. We just noticed that this actually contains a new tutorial. I will take a look next week at your tutorial!

File: tutorials/simple-ca-for-ssh/01.en.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: [submitter's name and email address here]
```
Comment (n-se) [inline tutorials/simple-ca-for-ssh/01.en.md:170]: Can you please sign off?

File: tutorials/simple-ca-for-ssh/01.de.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/simple-ca-for-ssh/de"
+slug: " Simple Certificate Authority for SSH"
+date: "2020-07-02"
+title: "Einrichtung einer SSH CA"
+short_description: "Um Benutzern den Zugang auf einen Server zeitlich zu begrenzen bietet sich der Einzatz einer CA an."
+tags: ["Administration", "SSH", "Security", "Certificate", "Authority"]
+author: "Paul Ludwig"
+author_link: https://github.com/pekab0
```
Comment (n-se) [inline tutorials/simple-ca-for-ssh/01.de.md:10]: Put this in qoutes " please - also in en tutorial

File: tutorials/simple-ca-for-ssh/01.de.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+author_img:
```
Comment (n-se) [inline tutorials/simple-ca-for-ssh/01.de.md:11]:
```suggestion
author_img: ""
```

Comment (n-se) [reply tutorials/simple-ca-for-ssh/01.de.md:11]: Also in en please

File: tutorials/simple-ca-for-ssh/01.de.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: [submitter's name and email address here]
```
Comment (n-se) [inline tutorials/simple-ca-for-ssh/01.de.md:174]: Can you please sign off?

**Comment by n-se** [comment]
Do you still plan to work on this tutorial?

**Comment by pekab0** [comment]
Hello.
Yes, i still work on this tutorial. Sorry for this delay, i was on vacation.

**Review by rook59 — APPROVED** [review]

**Pull request merged**